pages/dish/[id].js

```diff
@@ -1,15 +1,55 @@
 import { useRouter } from "next/router";
+import ReviewResult from "../../components/ReviewResult/ReviewResult";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Text, List, Group } from "@mantine/core";
+import styles from "./id.module.css"
 
 export default function ReviewsPage() {
 	const router = useRouter();
 	const { id, dish, diningHall, station } = router.query;
+	const [reviews, setReviews] = useState([]);
+	const [averageRating, setAverageRating] = useState(0);
+
+	useEffect(() => {
+		async function getReviews(params) {
+			const reviewsResponse = await axios.get(`/api/getReviews`, { params: { dish_id: id }})
+
+			const reviewResults = reviewsResponse.data.map(
+				(review) => ({
+					...review,
+				})
+			);
+			setReviews(reviewResults);
+		}
+
+		getReviews();
+		
+	}, [id]);
+
+	useEffect(() => {
+		if (reviews.length > 0) {
```

Comment:
Super small detail, but should set average rating to be NA or some default value in case there are no ratings for that dish? For instance, when I click on tomato bisque (which has no reviews), its rating is just the same as the rating of the previous dish I clicked on.